ext/File-Find/t/taint.t

```diff
@@ -87,22 +98,37 @@ is($found, 1, 'taint.t found once again');
 my $case = 2;
 my $FastFileTests_OK = 0;
 
-my $test_root_dir; # where we are when this test starts
-my $test_root_dir_tainted = cwd();
-if ($test_root_dir_tainted =~ /^(.*)$/) {
-    $test_root_dir = $1;
-} else {
-    die "Failed to untaint root dir of test";
+my $chdir_error = "";
+chdir($test_temp_dir)
+    or $chdir_error= "Failed to chdir to '$test_temp_dir': $!";
+is($chdir_error,"","chdir to temp dir '$test_temp_dir' successful")
+    or die $chdir_error;
+
+sub _something_wrong {
```

Comment:
sub `_something_wrong()` appears to be defined in each of two test files.  Could it be moved to `t/lib/Testing.pm` instead?

Reply:
Thanks. Good suggestion. Done. :-)

Reply:
And actually I went a step further and moved the identical code from both into Testing.pm as well. Good catch.